src/autocomplete/popup.js

```diff
@@ -408,6 +408,22 @@ class AcePopup {
         popup.$imageSize = 0;
         popup.$borderSize = 1;
 
+        /**
+         * Get HTML element containing the popup data.
+         * @returns {HTMLDivElement} element 
+         */
+        popup.getDataContainer = function() {
+            return this.renderer.$textLayer.element;
```

Comment:
Was there a reason for exposing `renderer.$textLayer.element` here and not, for example, `renderer.content` or `render.scroller`? (which encompass all layers of the popup and might allow more flexibility when styling the popup)